karmada/0.1.0/artifacthub-pkg.yaml

```diff
@@ -0,0 +1,27 @@
+version: 0.1.0
```

Comment:
U can move this file into the `karmada` folder. The `karmada/0.1.0/artifacthub-pkg.yaml` file will be generated automatically through CI.